webapp/lib/controller.dart

```diff
@@ -1448,6 +1479,9 @@ void showAndLogError(error, trace) {
     errMsg = "A network problem occurred: ${error.message}";
   } else if (error is Exception) {
     errMsg = "An internal error occurred: ${error.runtimeType}";
+  } else if (error is FirebaseError){
+    errMsg = "An firestore error occured: ${error.code} [${error.message}]";
+    view.bannerView.showBanner("You don't have access to any dataset. Please contact your project administrator");
```

Comment:
Phrasing nit:
```suggestion
    view.bannerView.showBanner("You don't have access to this dataset. Please contact your project administrator");
```
`FirebaseError` seems like quite a generic error type - is it used only for missing permissions errors? If not, is there another way we can isolate and show this message only for missing permission errors?

Reply:
It seems like it's a generic error and not specific to permission error as seen here [doc1](https://pub.dev/documentation/firebase/latest/firebase/FirebaseError-class.html), [doc2](https://firebase.google.com/docs/reference/js/firebase.FirebaseError). We'd probably have to come up with a better way.

Reply:
What do you suggest?